3drepo/3drepo.io
# ISSUE #3357 - Delete Project Modal

This fixes #3357 
- Added ability to delete projects
- Added confidence check to delete modals
- Added api/saga/redux for deleting a project
- Added saga and redux tests for deleting project

#### Test cases
- Delete a project
- View other delete modals (federation/container) to see that they are unaffected and don't require confirmation
